www/ui_components/add-server-view.html

```diff
@@ -157,7 +157,7 @@
         <div class="title">[[localize('server-add-access-key')]]</div>
         <div class="faded">[[localize('server-add-instructions')]]</div>
       </div>
-      <paper-input id="accessKeyInput" class="shadow" label="[[localize('server-access-key-label', 'ssProtocol', 'ss://')]]" no-label-float value={{accessKey}} pattern="^ss://[A-Za-z0-9+/=][#]?.*$">
+      <paper-input id="accessKeyInput" class="shadow" label="[[localize('server-access-key-label', 'ssProtocol', 'ss://')]]" no-label-float value={{accessKey}} pattern="(^ss://[A-Za-z0-9+/=][#]?.*$)|(^https://s3.amazonaws.com/outline-vpn/(index.html)|(invite.html).*#.*/invite/ss.*$)">
```

Comment:
It's probably fine as is, but you could add `ss://[A-Za-z0-9+/=][#]?.*$` to the new regex.

Reply:
Normally, the fragment is percent-encoded, but it's not clear to me whether that's actually a requirement.  I can tighten this regex but should I include support for both encoded and non-encoded fragments?  (BTW, the existing regex is not really working as intended: it's actually just checking the first character of the credentials, and the rest is `.*`.)